route_info_test.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/gobuffalo/buffalo/render"
 	"github.com/gobuffalo/httptest"
+	"github.com/pkg/errors"
```

Comment:
@dmuriel I suggest here using fmt.Errorf("msg: %w" to avoid the external dependency.